### PR TITLE
libsigc++: Update to 2.10.1

### PR DESCRIPTION
**Pull request by neheb**
Size goes from 15613 to 8427 bytes.

2.10.1 is the latest version. Version 2.99.x is the unstable series.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @sbyx 
Compile tested: ramips

libsigc++ is only used by libtorrent and rtorrent, both of which I maintain.

### CODE REVIEW

**Comment by neheb** [comment]
No idea how to fix CircleCI. It's failing on the ++

**Comment by neheb** [comment]
Looks like faking the package name works.

File: libs/libsigc++/Makefile [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=@GNOME/libsigc++/2.5
-PKG_HASH:=ecf55f53d6058ba6e41985b862f2e95fb5c2b31c008caa16984e790547337ea7
+PKG_SOURCE_URL:=@GNOME/libsigc++/2.10
+PKG_HASH:=c9a25f26178c6cbb147f9904d8c533b5a5c5111a41ac2eb781eb734eea446003
 PKG_MAINTAINER:=Steven Barth <cyrus@openwrt.org>
 PKG_LICENSE:=LGPL-2.1
 
-PKG_FIXUP:=autoreconf
```
Comment (thess) [inline libs/libsigc++/Makefile]: As discussed before, please do not remove PKG_FIXUP.  #6683, #6910, #7055, etc.

Comment (neheb) [reply libs/libsigc++/Makefile]: Filesize goes up to 8433

**Comment by thess** [comment]
If you want package ownership since it's a dependent of your other packages, I sure Steven would not mind. He's been MIA for quite a while now.

I'll see what can be done about the CircleCI script.

**Comment by neheb** [comment]
Switched maintainer.

**Pull request merged**